packages/@storefront/structure/src/carousel/index.ts

```diff
@@ -28,6 +28,11 @@ class Carousel {
 
   onMount() {
     utils.WINDOW().addEventListener('resize', this.forceUpdate);
+
+    // this forces an update on mount which in turn sets
+    // items to state from props by calling cloneItems
+    // so that items display within the carousel
+    this.set(true);
```

Comment:
Can you leave a comment as to why we need to use `this.set(true)` in the onMount? I think it's clear as to what it will do here, but more context on why it's here would be good.